spec/acceptance/steps/open_search_steps.rb

```diff
@@ -15,6 +15,7 @@ module OpenSearchSteps
 
   step 'I request the open search description document' do
     osdd_url = "http://#{@valid_envs[@target_env]['Hostname']}/#{@valid_envs[@target_env]['Path']}"
+    puts "OSDD_URL: #{osdd_url}"
```

Comment:
Is this here for debugging, or necessary to get the test to run?

Reply:
Debugging.  I've removed it :)